contrib/index.html

```diff
@@ -8,174 +8,211 @@
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-    <header>
-      <h1>
-        Hey there! Thanks for contributing to
-        <a href="http://bitters.bourbon.io">Bitters</a>
-      </h1>
-    </header>
-    <main>
-        <p>
-          The purpose of this HTML is to help contibutors test and develop
-          Bitters styles and default settings.
-        </p>
-        <hr>
-        <h1 id="headings">Headings</h1>
-        <h1>Heading 1</h1>
-        <h2>Heading 2</h2>
-        <h3>Heading 3</h3>
-        <h4>Heading 4</h4>
-        <h5>Heading 5</h5>
-        <h6>Heading 6</h6>
-        <hr>
-        <h1 id="paragraph">Paragraph</h1>
+    <header class="welcome-message" role="banner">
+      <div class="container">
+        <h1>
+          Hey there! Thanks for contributing to
+          <a href="http://bitters.bourbon.io">Bitters</a>
+        </h1>
         <p>
-          Lorem ipsum dolor sit amet, <a href="#" title="test link">test link</a>
-          adipiscing elit. Nullam dignissim convallis est. Quisque aliquam. Donec
-          faucibus. Nunc iaculis suscipit dui. Nam sit amet sem. Aliquam libero
-          nisi, imperdiet at, tincidunt nec, gravida vehicula, nisl. Praesent
-          mattis, massa quis luctus fermentum, turpis mi volutpat justo, eu
-          volutpat enim diam eget metus. Maecenas ornare tortor. Donec sed tellus
-          eget sapien fringilla nonummy. Mauris a ante. Suspendisse quam sem,
-          consequat at, commodo vitae, feugiat in, nunc. Morbi imperdiet augue
-          quis tellus.
+          The purpose of this page is to help contributors view and test
+          changes to Bitters’ styles.
```

Comment:
This is weird but the correct way to write this is "Bitters's because the 's' in Bitters is not a pluralization. Although you could make the argument that since Bitters is a proper noun, an extra 's' is not required.

I feel very conflicted. :expressionless: